docs/localization/LOCALIZATION.md

```diff
@@ -74,343 +78,465 @@ export function localizationConfigurationFactory() {
     this.localizationService.useLanguage(language);
   }
 }
 ```
 
 Now we can start using pipes:
 
-```typescript
-
-<p>{{today | date:'fullDate': '': localizationService.getCurrentLanguage()}}</p>
-
-<p>{{1.5487 | number: '': localizationService.getCurrentLanguage()}}</p>
+```html
+<p>{{today | date: 'fullDate' : '' : localizationService.getCurrentLanguage()}}</p>
 
-<p>{{1.5487 | currency:'EUR':'symbol':'':localizationService.getCurrentLanguage()}}</p>
+<p>{{1.5487 | number: '' : localizationService.getCurrentLanguage()}}</p>
 
+<p>{{1.5487 | currency: 'EUR' : 'symbol' : '' : localizationService.getCurrentLanguage()}}</p>
 ```
 
 ### How to add custom translations in the container
 
-By default, a container needs to provide an override of translation keys to the presenter:
+A container can have its own translations (for error messages for example). In this case, the container becomes `Translatable` as follows:
 
 ```typescript
-import {MyPresTranslation} from '../presenter/my.translation.ts';
+import { Translatable } from '@o3r/localization';
+import { MyContTranslation } from '../my.translation.ts';
 
 @Component({
-  selector: 'o3r-forms-poc-cont',
-  templateUrl: './forms-poc-cont.template.html',
+  selector: 'o3r-my-cont',
+  templateUrl: './my.template.html',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MyContComponent implements FormsPocContContext, Block {
+export class MyContComponent implements Translatable<MyContTranslation>, Block {
 
   /**
    * Localization of the component
    */
   @Input()
-  public translations: Partial<MyPresTranslation>;
-
-  ...
-
-  public getMyPresContext(overrideContext) {
-    return {
-      translations,
-      inputs: {
-        // ...
-      }
-    };
-  }
+  @Localization('./my.localization.json')
+  public translations: MyContTranslation;
 }
 ```
 
-A container can have its own translation (for error messages for example). In this case the container become `Translatable` as following:
+There is also a possibility for the container to override the presenter's translations and pass them as input to the presenter, for example:
 
 ```typescript
-import {MyPresTranslation} from '../presenter/my.translation.ts';
-import {MyContTranslation} from '../my.translation.ts';
-import {Translatable} from '@o3r/localization';
+import { MyPresTranslation } from '../presenter/my.translation.ts';
 
 @Component({
   selector: 'o3r-my-cont',
   templateUrl: './my.template.html',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MyContComponent implements FormsPocContContext, Translatable<MyContTranslation>, Block {
+export class MyContComponent implements MyContContext, Block {
 
   /**
    * Localization of the component
    */
   @Input()
-  @Localization('./my.localization.json')
-  public translations: Partial<MyPresTranslation> & MyContTranslation;
+  public translations: Partial<MyPresTranslation>;
 
-  ...
+  // ...
 
   public getMyPresContext(overrideContext) {
```

Comment:
if I am not mistaken this method is deprecated, the example seems to be from the time when all the components have container/presenter structure. IMO you can remove the code example and let only the explanation, it's quite enough.